README.md

```diff
@@ -1 +1 @@
-# reddit-bot
\ No newline at end of file
+# Reddit autopost for up-for-grabs > /r/RespositoryRequest
```

Comment:
Typo: Re<b>s</b>pository

Maybe link to https://www.reddit.com/r/RepositoryRequest, too?